apps/hobbies-helsinki/src/hooks/useRHHCConfig.tsx

```diff
@@ -109,6 +123,22 @@ export default function useRHHCConfig() {
       },
       utils: {
         ...rhhcDefaultConfig.utils,
+        getRoutedInternalHref: (
+          link: string,
+          _type: ModuleItemTypeEnum
+        ): string => {
+          if (_type === ModuleItemTypeEnum.Venue) {
+            // quick fix for venue url rewrites
+            return getLocalizedCmsItemUrl(
+              ROUTES.VENUES,
+              {
+                venueId: getVenueSourceId(link),
+              },
+              locale
+            );
+          }
+          return link || '#';
+        },
```

Comment:
Our deprecated feature is in use again 😄

Reply:
yup. temporary. Jaakko cannot provide the fix now